client/layout/index.jsx

```diff
@@ -162,6 +168,14 @@ class Layout extends Component {
 				}
 			}
 		}
+
+		if ( this.props.isStateRandomlyCleared ) {
```

Comment:
Is there any chance we can abstract that to a separate component that we conditionally load only on the environments where sympathy mode is enabled? The layout component is quite huge already so anything that can be abstracted away from it is a small win.

Reply:
Good idea I will give this a go.